src/verifications/verifications.ts

```diff
@@ -1753,8 +1835,14 @@ export class Verifications extends Logger {
     };
 
     // run tx
-    const [executionId, blockNumber] = await this.options.executor.executeContractTransaction(
-      userIdentity, 'execute', options, this.contracts.registry.options.address, 0, data);
+    const [executionId, blockNumber] = await (this.options.executor.executeContractTransaction as any)(...[
```

Comment:
what happens here?????

Reply:
it appends signedTransactionInfo or not :)

Reply:
- when not converting function to a generic function, linter could not check argument count properly (function uses  rest argument but counting failed), therefore the `any` (maybe `Function` would also be usable here)
- `executeDelegated` needs 1 argument more than `execute`, the `signedTransactionInfo`, see [here](https://github.com/evannetwork/smart-contracts-core/blob/develop/contracts/verifications/KeyHolder.sol#L76)
- as the entire function call is converted into a solidity transaction, giving arguments, that are not required as `undefined` or `null` has to be prevented, as it would mess up the mapping to the smart contract function
- `myFunction(...[requiredArguments].concat(doINeedOneMoreArgument ? [optionalArgument] : []))` calls `myFunction` with a different number of arguments, depending on `doINeedOneMoreArgument` and prevents giving `myFunction` an extra `undefined` argument.